docs/dotnetcore/introducing-dotnetcore/introducing-dotnetcore.rst

```diff
@@ -0,0 +1,94 @@
+
+.. _`.NET Framework Client Profile`: https://msdn.microsoft.com/en-us/library/cc656912%28v=vs.110%29.aspx
+
+In addition to being able to target a variety of different device platforms, there was also pressure from the server side to reduce the overall footprint of the .NET framework. By factoring the CoreFX libraries and minimizing the size of CoreFX, server-based applications built with ASP.NET 5 can minimize their size, allowing cloud-based hosting environments to increase the density of applications they can support on a given set of hardware resources. This will provide increased economies of scale for ASP.NET applications hosted on Azure, for instance.
+
+.NET Core and ASP.NET
+^^^^^^^^^^^^^^^^^^^^^
+
+ASP.NET 5 can target either the full .NET Framework or .NET Core. In fact, ASP.NET 5 projects can be cross-compiled, targeting both of these frameworks in a single project, and this is how the project templates ship with Visual Studio 2015. For example, the *frameworks* section of *project.json* in a new ASP.NET 5 web project will target *dnx451* and *dnxcore50* by default:
+
+.. code-block:: javascript
+
+	"frameworks": {
+		"dnx451": { },
+		"dnxcore50": { }
+	},
+
+*dnx451* is the full .NET Framework; *dnxcore50* is .NET Core 5 (5.0).
+
+By contrast, earlier versions of ASP.NET can only target the .NET Framework
+ASP.NET 4.6 and earlier can only target the .NET Framework. Since both the full .NET Framework and .NET Core can both be targeted at the same time by ASP.NET 5, the recommendation is to target both with new applications, resolving issues with dependencies that are incompatible with .NET Core through the use of conditional compilation directives or choosing to require the full .NET Framework, if necessary.
+
+.. note:: You can use compiler directives (**#if**) to check for symbols that correspond to the two frameworks: **DNX451** and **DNXCORE50**.
+	
+If for instance you have code that uses resources that are not available as part of .NET Core, you can surround them in a conditional compilation directive:
+
+.. code-block:: c#
+
+	#if DNX451
+		// utilize resource only available with full .NET
+	#endif
+
+If you want to only target .NET Core, you should remove *dnx451* from the *frameworks* listed in *project.json*.
+
+.NET Core and NuGet
+^^^^^^^^^^^^^^^^^^^
+
+As part of the effort to *factor* the CoreFX, the individual assemblies are available as separate NuGet packages. In fact, NuGet is the primary delivery vehicle for .NET Core. If, for example, you need to use immutable collections, you can install the System.Collections.Immutable package via NuGet. The NuGet version will also align with the assembly version, and will use `semantic versioning <http://semver.org>`_.
```

Comment:
A nit, but it isn't so much that the assemblies are 'available' as NuGet packages, but that they are packaged as NuGet packages and distributed on NuGet.org. They are not 'available' a different way.